.clang-tidy

```
@@ -63,6 +63,9 @@ Checks: "-*,
 HeaderFilterRegex: '\/modules\/'
 ExcludeHeaderFilterRegex: ".pb.h$"
 
+ExtraArgs:
+  - -Wno-unused-command-line-argument
```

Need to add this, as otherwise with the new clang-tidy you won't get errors

```suggestion
  - -Wno-unused-command-line-argument
  
  WarningsAsErrors: "*"
```